front/src/actions/dashboard/boxes/devices.js

```diff
@@ -0,0 +1,164 @@
+import { RequestStatus } from '../../../utils/consts';
+import createBoxActions from '../boxActions';
+import createDeviceActions from '../../device';
+import update from 'immutability-helper';
+import get from 'get-value';
+import debounce from 'debounce';
+
+const BOX_KEY = 'Devices';
+
+function createActions(store) {
```

Comment:
We try to avoid using unistore for that now, since you're rewriting this component, maybe can you build this component using only class functions ? ( Cf: https://community.gladysassistant.com/t/frontend-amelioration-diverses-experience-developpement/8125 )

Reply:
Removed :)